test/E2E/E2ETest.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Xunit;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.ProjectModel;
+
+namespace ConsoleApplication
+{
+    public class E2ETest
+    {
+        private static readonly string EXPECTED_OUTPUT = "Hello World!";
+        private static readonly string TEST_PROJECT_NAME = "hellotest";
+        private static readonly string OUTPUT_FOLDER_NAME = "testbin";
+
+        public static void Main()
+        {
+            Console.WriteLine("Dummy Entrypoint.");
+        }
+
+        [Fact]
+        public static void TestE2E()
+        {
```

Comment:
This is a simple refactoring which can have many benefits. Running the E2E I need to know all the commands that are broken instead of the first one only.